js/views/userPage/Follow.js

```diff
@@ -40,6 +40,10 @@ export default class extends BaseVw {
         this.render();
       });
     }
+
+    this.listenTo(this.followCol, 'update', (col) => {
```

Comment:
This only works if we're not going to paginate the collection, which I believe we're going to do for at least one of the collections, right? (we really should paginate both, but that's a separate issue)

For any collection we paginate, we will need to rely on the server count. The user won't know it's off and if and when we fully fetch the collection, we could adjust the count then.